src/components/hv-picker-field/field-label/index.tsx

```diff
@@ -6,7 +6,7 @@
  *
  */
 
-import type { Props } from './types';
+import type {Props} from './types';
```

Comment:
I wonder what causes these changes - a newly eslint rule on the repo, or a IDE override.